docs/arch/adr-001.md

```diff
@@ -0,0 +1,38 @@
+# ADR-1: Code Generation
+
+## Context
+
+Serializing and deserializing JSON in Java works best when performing it to/from POJOs.
```

Comment:
Define POJO.
`to/from POJOs (plain old java objects)` or something similar, people reading may not understand this.

Reply:
I probably will. But ADRs are not intended to target those new to ErisCasper or serve as an introduction.

Reply:
Yeah I'm aware, still good to have an in-line interpretation of the abbrev you're using.